packages/runtime/src/useCases/consumption/attributes/GetSharedVersionsOfAttribute.ts

```diff
@@ -30,7 +30,7 @@ export class GetSharedVersionsOfRepositoryAttributeUseCase extends UseCase<GetSh
         super(validator);
     }
 
-    protected async executeInternal(request: GetSharedVersionsOfRepositoryAttributeRequest): Promise<Result<LocalAttributeDTO[]>> {
+    protected async executeInternal(request: GetSharedVersionsOfAttributeRequest): Promise<Result<LocalAttributeDTO[]>> {
```

Comment:
you have renamed the UseCase, but you still check if it's a repository attribute